### DISCUSSION TITLE
What is the relation between EDC & Catena-X?

### DISCUSSION THREAD

**Discussion by dilip-dhankecha** [eclipse-edc/.github#39] [Q&A] [answered]
If any small enterprise having data and they will store those data to cloud.
Now they want to exchange the data with other SME so they wants to introduce a EDC.
So where the EDC will be held?
Or i can say this will be in the application or EDC directly connect over the Data cloud storage?

Now suppose Small enterprise need to exchange data with multiple enterprise then where EDC will be held and where the catena-x components(like identity Broker, DAPS, clearing house etc.) will be held?

Is catena-x will be with in EDC or it will be other component which will connected with EDC?
Basically how it looks like?
And How data exchange will be occured?

**Comment by mspiekermann** [comment, marked as answer]
Hi @dilipdhankecha2530,
the basic idea for a data space is the availability of at leat one connector component for each participant. This could be either self-hosted or through "Connector as a Service" offerings on behalf of the client(s). These EDC instances will then connect to other services, the data space provides. 
Depending on the  data space authority, you can think of various architectural patterns from completely decentralized to central shared services. 
In your example of IDSA-DAPS, Broker, ClearingHouse etc. it would built on the existing APIs and IDS-messages using the corresponding IDS-extensions.

> Basically how it looks like?

Have a look at the graphic of https://eclipse-dataspaceconnector.github.io/DataSpaceConnector/#/README
